### PR TITLE
Code changes after code review

### PR DESCRIPTION
**Pull request by Buddhiprabha**
I have made changes according to Shruti's code review comments.

Also I have added code for Staff Contact of reporting(Dialer and SMS apps)

### CODE REVIEW

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 1]
```diff
@@ -41,14 +45,39 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
-        contact_pcmo = (Button) findViewById(R.id.post_staff_pcmo);
-        contact_ssm = (Button) findViewById(R.id.post_staff_ssm);
-        contact_sarl = (Button) findViewById(R.id.post_staff_sarl);
-        current_location = (TextView) findViewById(R.id.post_staff_current_location);
+        locationDetails.put(getResources().getString(R.string.loc1_name), new LocationDetails(getResources().getString(R.string.loc1_name), getResources().getString(R.string.loc1_pcmo), getResources().getString(R.string.loc1_ssm), getResources().getString(R.string.loc1_sarl)));
+        locationDetails.put(getResources().getString(R.string.loc2_name), new LocationDetails(getResources().getString(R.string.loc2_name), getResources().getString(R.string.loc2_pcmo), getResources().getString(R.string.loc2_ssm), getResources().getString(R.string.loc2_sarl)));
+        locationDetails.put(getResources().getString(R.string.loc3_name), new LocationDetails(getResources().getString(R.string.loc3_name), getResources().getString(R.string.loc3_pcmo), getResources().getString(R.string.loc3_ssm), getResources().getString(R.string.loc3_sarl)));
+
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: I don't understand why these locations are constant strings. Should they not be populated from real time data?

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java [hunk 2]
```diff
@@ -58,13 +87,51 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
+    /**
+     * Creates a Dialog for the user to choose Dialer app or SMS app
+     * @param v the view clicked
+     * @param numberToContact contact number corresponding to the view
+     */
+    private void createDialog(View v, final String numberToContact){
+        DialogInterface.OnClickListener dialogClickListener = new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
```
Comment (shruti-gupta) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: Which? Not a self explanatory name

File: app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java (hunk 1 above)
Comment (Buddhiprabha) [inline app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java]: We are not getting data from a server at the moment as discussed. Therefore, data is added to XML and being retrieved from there :)

**Comment by rosariorobinson** [comment]
Update?

**Comment by Buddhiprabha** [comment]
The changes were done and the code was committed with updates

**Pull request merged**